torchvision/models/detection/roi_heads.py

```diff
@@ -73,7 +74,11 @@ def maskrcnn_inference(x, labels):
     index = torch.arange(num_masks, device=labels.device)
     mask_prob = mask_prob[index, labels][:, None]
 
-    mask_prob = mask_prob.split(boxes_per_image, dim=0)
+    if torchvision._is_tracing():
```

Comment:
Can you make the condition be instead `if len(boxes_per_image) == 1`, as you did in #1401 ?